src/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring/Windows/Interop/MemoryInfo.cs

```diff
@@ -10,20 +10,16 @@ namespace Microsoft.Extensions.Diagnostics.ResourceMonitoring.Windows.Interop;
 /// Native memory interop methods.
 /// </summary>
 [ExcludeFromCodeCoverage]
-internal sealed class MemoryInfo : IMemoryInfo
+internal sealed partial class MemoryInfo : IMemoryInfo
 {
-    internal MemoryInfo()
-    {
-    }
-
     /// <summary>
     /// Get the memory status of the host.
     /// </summary>
     /// <returns>Memory status information.</returns>
-    public unsafe MEMORYSTATUSEX GetMemoryStatus()
+    public MEMORYSTATUSEX GetMemoryStatus()
```

Comment:
This method should look similar to
https://github.com/dotnet/runtime/blob/31733b9a35185785427bac69ef80a4eb56b727c2/src/libraries/System.IO.MemoryMappedFiles/src/System/IO/MemoryMappedFiles/Interop.Windows.cs#L12-L24

Reply:
Updated